src/main/java/org/codeforamerica/shiba/mnit/CountyMapConfiguration.java

```diff
@@ -785,7 +785,10 @@ private void initializeDefaultCountyMap() {
     addCounty(Fillmore, "507-765-2175");
     addCounty(Freeborn, "507-377-5400");
     addCounty(Goodhue, "651-385-3200");
-    addCounty(Grant, "800-291-2827");
+    addCounty(Grant, "320-634-7758", //Grant and Pope share same phone number and NPI
+    		"help+dev@mnbenefits.org",
+            "5a869a7f-4ebb-44ae-b2db-a6f40396d6a0",
+            "A000061200");
```

Comment:
@EricMN35Johnson The prod mapping for Grant is different than Pope. Should they be the same?
```
        "6498cbd4-d283-40c2-b3ba-9fb9153546fb",
        "1225119381");
```